src/test/java/com/github/tomakehurst/wiremock/LogTimingAcceptanceTest.java

```diff
@@ -54,12 +54,12 @@ public void serveEventIncludesTotalAndServeDuration() {
 
   @Test
   public void includesAddedDelayInTotalWhenAsync() {
-    final int DELAY = 500;
+    final int delay = 500;
```

Comment:
We should probably follow up and remove the `final`s from local varialbes, as I don't think we want them

Reply:
Done; made the diff quite a bit larger, with more potential for conflicts, but should be OK if merged soon-ish.